### PR TITLE
Bug/153 multiple mapped input columns property

### PR DESCRIPTION
**Pull request by jhorcicka**
Fixes https://github.com/NCIM/datacleaner-commercial-editions/issues/153

MultipleMappedInputColumnsPropertyWidget was not updated correctly when a component (which columns were used in the widget) was removed from the graph.

### CODE REVIEW

**Comment by ClaudiaPHI** [comment]
LGTM! I have tested it and it seems to work fine.

File: desktop/ui/src/main/java/org/datacleaner/widgets/properties/MultipleMappedStringsPropertyWidget.java [hunk 1]
```diff
@@ -215,6 +204,23 @@ public MappedStringsPropertyWidget getMappedStringsPropertyWidget() {
         }
         return result.toArray(new InputColumn[result.size()]);
     }
+    
+    public void updateMappedStrings() {
+        if (_alreadyUpdating) {
+            return;
+        }
+        
+        SwingUtilities.invokeLater(new Runnable() {
```
Comment (LosD) [inline desktop/ui/src/main/java/org/datacleaner/widgets/properties/MultipleMappedStringsPropertyWidget.java]: Could be replaced with lambda

**Comment by LosD** [comment]
LGTMtoo, but you could replace a single variable with a lambda.

**Comment by jhorcicka** [comment]
@LosD, updated.

**Comment by LosD** [comment]
LGTM. Travis was happy before, and probably still is, but let's wait for it to be happy again, then merge :)

**Pull request merged**